actions/serverless_branch_deploy/action.yml

```diff
@@ -77,14 +77,18 @@ runs:
         env_vars: ${{ inputs.env_vars }}
         base_image: ${{ inputs.base_image }}
 
+    - name: Generate image tag suffix
+      id: generate-suffix
+      uses: ./action-repo/actions/utils/generate_unique_suffix
+
     - name: Build and push Docker image
       if: ${{ github.event.pull_request.state != 'closed' }}
       uses: docker/build-push-action@v3
       with:
         ssh: ${{ env.DOCKER_BUILD_SSH }}
         context: ${{ fromJSON(inputs.location).directory }}
         push: true
-        tags: "${{ env.REGISTRY_URL }}:prod-${{ fromJson(inputs.location).name }}-${{ github.sha }}"
+        tags: "${{ env.REGISTRY_URL }}:branch-${{ fromJson(inputs.location).name }}-${{ github.sha }}-${{ steps.generate-suffix.outputs.suffix }}"
```

Comment:
this was previously (confusingly) called prod, i assume to take advantage of caching if it happened to match up with an existing SHA? no need for that anymore